.github/workflows/integrations.yml

```diff
@@ -66,13 +66,12 @@ jobs:
         with:
           repository: OpenAssetIO/OpenAssetIO-Manager-BAL
           path: external/BAL
+          ref: feature/relatedReferencesRefactor
```

Comment:
[consider] If I understand correctly, this `drop!` commit will not be dropped immediately, but will be dropped when the feature branch is merged to `main`?

In which case, when this commit is dropped, it will revert to disabling BAL tests.  

Shouldn't the commit be split into two, a normal commit to re-enable the tests, and a `drop!` commit to add the `ref:`?

Reply:
Yeah - a great Q - I kind of left it as that to make sure we schemed on it. The one that disables CI is also a `drop!`.

Reply:
The BAL integration test was disabled in f2947172, which isn't a drop! commit.

Reply:
Ahh yeah- sorry I thought it was - I might have combined it before after all